spec/faraday/adapter_registry_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+RSpec.describe Faraday::AdapterRegistry do
+  describe '.initialize' do
+    subject { described_class.new }
```

Comment:
```suggestion
    subject(:registry) { described_class.new }
```

Reply:
Good idea. There should be a cop for that :)

Reply:
`rubocop_rspec` is a bag of such cops. They're pretty good at making one refactor tests.